tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,167 @@
+					deleteInstanceIds = append(deleteInstanceIds, instance.InstanceId)
+				}
+			}
+		}
+		if describeInstancesOutput.NextToken == nil {
+			break
+		}
+		nextToken = describeInstancesOutput.NextToken
+	}
+
+	if len(deleteInstanceIds) < 1 {
+		log.Printf("No instances to delete")
+		return nil
+	}
+
+	terminateInstancesInput := ec2.TerminateInstancesInput{InstanceIds: deleteInstanceIds}
+	_, terminateInstancesErr := ec2client.TerminateInstances(&terminateInstancesInput)
+	if terminateInstancesErr != nil {
+		log.Printf("Failed to terminate instances %v because of %v",deleteInstanceIds, terminateInstancesErr)
+		return terminateInstancesErr
+	}
+
+	return nil
+}
+
+func destroyLoadBalancerResource()(error){
+	// Set up aws go sdk session
+	//Enable default region and credentials
+	//Documents: https://docs.aws.amazon.com/ja_jp/sdk-for-go/v1/developer-guide/configuring-sdk.html
+	session := session.Must(
+		session.NewSessionWithOptions(
+			session.Options{
+				SharedConfigState: session.SharedConfigEnable,
+			}))
+
+	svc := elbv2.New(session)
+
+	//ELB Go SDK currently does not support filter tag or filter wildcard. Only supports with matching name
+	//Documentation: https://github.com/aws/aws-sdk-go/blob/02266ed24221ac21bb37d6ac614d1ced95407556/service/elbv2/api.go#L5879-L5895
+
+	describeLoadBalancerInputs := &elbv2.DescribeLoadBalancersInput{}
+
+
+	describeLoadBalancerOutputs, describeLoadBalancerErr := svc.DescribeLoadBalancers(describeLoadBalancerInputs)
+
+	if describeLoadBalancerErr != nil {
+		log.Printf("Failed to get metadata for load balancer because of %v", describeLoadBalancerErr)
+		return describeLoadBalancerErr
+	}
+
+	for _, lb := range describeLoadBalancerOutputs.LoadBalancers {
+
+		//Skipping lb that does not contain aoc-lb string (relating to aws-otel-test-framework)
+		if filterLbNameResult := strings.Contains(*lb.LoadBalancerName, containLbString); !filterLbNameResult {
+			log.Printf("Skipping lb %s because of not containing %s", *lb.LoadBalancerName, containLbString)
+			continue
+		}
+
+		//Skipping lb that does not older than 5 days
+		if !time.Now().UTC().Add(-1 * time.Hour * 24 * pastDayDelete).After(*lb.CreatedTime) {
+			log.Printf("Skipping lb %s with launch-date %v", *lb.LoadBalancerName, lb.CreatedTime)
+			continue
+		}
+
+		log.Printf("Trying to delete lb %s with launch-date %v", *lb.LoadBalancerName, lb.CreatedTime)
+
+		//Delete load balancer
+		//Documentation: https://github.com/aws/aws-sdk-go/blob/main/service/elbv2/api.go#L829-L844
+		deleteLoadBalancerInput := &elbv2.DeleteLoadBalancerInput{
+			LoadBalancerArn: lb.LoadBalancerArn,
+		}
+		_, deleteLoadBalancerError := svc.DeleteLoadBalancer(deleteLoadBalancerInput)
+
+		if deleteLoadBalancerError != nil {
+			log.Printf("Failed to delete lb %s because of %v", *lb.LoadBalancerName,deleteLoadBalancerError)
+			return deleteLoadBalancerError
+		}
+
+		log.Printf("Delete lb %s successfully", *lb.LoadBalancerName)
+
+
+	}
+
+	return nil
+
```

Comment:
nitpick: Remove the extra spacing everywhere and run the formatter. It's fine to have some, but I try to avoid leading and trailing spacing.

Reply:
Reasonable. Deleting some extra space.